### DISCUSSION TITLE
overview 是不是可以加个二维码？

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#615] [Ideas]
为什么有这个想法呢……是因为信息卡片的信息还是不太全，有时候需要看更多信息的时候就挺麻烦的（浏览器-Google搜索GHW-切换中文-切换角色-找到角色-点进去），信息卡片补全所有信息也不现实，所以是不是可以链接到 GHW 或者 BWIKI 或者米游社观测枢，需要更多信息的时候就长按跳转过去。
不过，可能不太美观就是了？

**Comment by Arondight** [comment]
虽然但是，这个功能感觉完全没有用……

**Comment by mark9804** [comment]
草，内鬼网我们自己用用不要声张，到时候会惹麻烦的

**Comment by mark9804** [comment]
加二维码也可以，加我们项目或者 silvery 项目的主页的二维码

**Reply by Arondight** [reply]
我感觉算了吧，本来这个项目的最初目的也是为了自用，因为我的想法原作者不关心的嘛，现在目的达到了，至少能较好的服务我们俩人了，也没啥必要宣传了，毕竟用的人越多越麻烦
